mobile/library/common/jni/jni_utility.cc

```diff
@@ -66,6 +66,16 @@ void jni_delete_const_global_ref(const void* context) {
   jni_delete_global_ref(const_cast<void*>(context));
 }
 
+bool exception_check(JNIEnv* env) {
```

Comment:
I'd be inclined to ABSL_MUST_USE_RESULT for this.  or at least make it clear we're clearing

Reply:
I do not think that it always have to be handled explicitly. Per ur suggestion, will change method's name to imply that it clears any pending exceptions